src/codecs/music_gme.c

```diff
@@ -0,0 +1,469 @@
+/*
+  SDL_mixer:  An audio mixer library based on the SDL library
+  Copyright (C) 1997-2020 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifdef MUSIC_GME
+
+#include "SDL_loadso.h"
+
+#include "music_gme.h"
+
+#include <gme/gme.h>
```

Comment:
Why I used <gme.h> than <gme/gme.h>?
- In some systems with older GME versions, the header was installed at the "include" directory directly and wasn't included into sub-directory
- Since some versions, the header started to be placed at the "gme" sub-directory because new public headers got been added by Michael Pyne (for internal needs of GME itself). And so, at the configure script you should detect the presence of the "gme" directory and attach it to the include path.

Reply:
The oldest package I found for fedora was from Aug. 2011 for Fedora-16:
http://archives.fedoraproject.org/pub/archive/fedora/linux/releases/16/Everything/i386/os/Packages/game-music-emu-devel-0.5.5-1.fc16.i686.rpm
It installs gme headers under `gme` directory, so we should be safe here

Reply:
Fine